### PR TITLE
chore: reprovision

### PR DESCRIPTION
**Pull request by danielmoll**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,52 +112,59 @@
...
-    "isparta": "^3.5.3",
+    "isparta": "^4.0.0",
     "istanbul": "^0.4.3",
     "karma": "^0.13.22",
-    "karma-browserify": "^5.0.2",
+    "karma-browserify": "^5.0.3",
     "karma-coverage": "^0.5.5",
-    "karma-mocha": "^1.0.1",
-    "karma-mocha-reporter": "^2.0.0",
+    "karma-mocha": "^0.2.2",
```
Comment (gcedo) [inline package.json]: Version downgrade.

**Comment by danielmoll** [comment]
@gcedo happy to merge?

**Comment by gcedo** [comment]
👍 LGTM

File: .travis.yml [hunk 2]
```diff
@@ -6,4 +6,6 @@ cache:
 node_js:
   - '4'
   - stable
-after_success: 'travis-after-all && npm run semantic-release && npm run pages -- --repo https://${GH_TOKEN}@github.com/economist-components/component-picture.git'
+after_success: >-
+  travis-after-all && npm run semantic-release && npm run pages -- --repo
+  https://${GH_TOKEN}@github.com/economist-components/component-picture.git
```
Comment (fabiosantoscode) [inline .travis.yml]: It's weird that this works without a leading backslash on line 10

Comment (gcedo) [inline .travis.yml]: Looks like `>-` is a line folding operator.

**Pull request merged**